media/css/readthedocs-doc-embed.css

```diff
@@ -152,3 +152,12 @@ div.rtd-pro.alabaster div.rtd-pro-about i.fa-info-circle:before {
         border-color: #656565;
     }
 }
+
+div.rtd-pro > p {
+    margin-top: 1em;
+    color: #838383;
+    font-size: 12px;
+    font-style: italic;
+    font-family: "Courier";
```

Comment:
Would be curious on feedback here. Want something to differentiate it from the ad visually.

Reply:
I think it is important to mention that RTD's ads are different from other people's so I like this.

The main part I'd reconsider is hardcoding a font. You might also consider not hardcoding the color but that's less critical. Since this will get displayed on multiple different themes (at least RTD and alabaster) you want the text to stick with the theme and not have the text stick out like a sore thumb to typophiles. I think not changing the font but just making it small, italic, and visually separated would be fine.

Perhaps `<p><small><em>Ads served ethically?</em></small></p>`?

Reply:
Yup, second on font-family change -- there are a lot of Font Things happening, which is good to avoid. The swap to a monospace font is a bit jarring. I think small and italic works just fine, I don't think we have a styling as part of the theme that makes sense to reuse here.